demo/demo.md

```diff
@@ -22,17 +22,24 @@ In these examples the structure of the HTML uses the slot element to pull the tr
 
 <div class="exampleWrapper">
   <auro-popover for="button1">
-    Top popover content!
+    <div slot="tooltip">Top popover content!</div>
```

Comment:
This is not required. 

Return the template to use `role` and then there is no requirement to wrap this string in a slot reference.

Reply:
Reverted.